.github/workflows/prs.yaml

```diff
@@ -37,14 +34,11 @@ jobs:
                 ls -l tools/
                 echo ""
                 echo "${FILE} does not have the correct permissions."
-                echo "Ensure are tools are executable by everyone and try again."
+                echo "Ensure all tools are executable by everyone and try again."
                 exit 1
               fi
             done
           done
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.10"
```

Comment:
We only need Python here to run the test code, which is just `pytest` + the standard library. Whatever version happens to be pre-installed on the `ubuntu-latest` runner from GitHub ([3.12.3, as of this writing](https://github.com/actions/runner-images/blob/25b33392eaa1122a494597b09ecfddd8c3cd1fa1/images/ubuntu/Ubuntu2404-Readme.md#L26)) should be fine.